app/static/src/store/actions.js

```diff
@@ -27,6 +27,7 @@ export function fetchEmployees() {
                 error => console.error(error)
             )
             .then(json => {
+                console.log(json)
```

Comment:
please remove this otherwise the log is polluted with JSONs that don't provide any information